.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
-    "files.trimTrailingWhitespace": false,
+    "files.trimTrailingWhitespace": true,
+	"[markdown]": {
+		"files.trimTrailingWhitespace": false
```

Comment:
Why disable this for markdown files?

Reply:
Markdown files use two spaces at the end of the line to represent a new line. Without this, vscode will remove those spaces.

Reply:
Ah, good point.